src/test/java/edu/unc/lib/boxc/migration/cdm/services/PostMigrationReportServiceTest.java

```diff
@@ -74,7 +74,7 @@ public void addSingleItemTest() throws Exception {
         assertContainsRow(rows, "25",
                 "http://localhost/cdm/singleitem/collection/proj/id/25",
                 "Work",
-                "http://localhost:46888/bxc/record/" + BOXC_ID_1,
+                "http://localhost:46887/bxc/record/" + BOXC_ID_1,
```

Comment:
can you make all these `"http://localhost:46887/bxc/record/"` into a constant like `BASE_URL` or something? might be more readable with what's actually different. and if the port changes in the future then it would be easier to update. could use the helper too if you wanted

Reply:
updated

Reply:
Updated